Makefile

```diff
@@ -3,11 +3,41 @@ MAINPACKAGE=./cmd/manager
 GOENV=GOOS=linux GOARCH=amd64 CGO_ENABLED=0
 GOFLAGS=-gcflags="all=-trimpath=${GOPATH}" -asmflags="all=-trimpath=${GOPATH}"
 
+OPERATORNAME := dedicated-admin-operator
+
+# Where do CSVs end up? Note, the CSV generator creates a subdirectory,
+# called $(FINALCSVDIR)
+CSVBASEDIR := csvs
+# FINALCSVDIR is created by the Python script
+FINALCSVDIR := $(CSVBASEDIR)/$(OPERATORNAME)
+
+CATALOG_SOURCEIMG := quay.io/redhat/$(OPERATORNAME)-cs
```

Comment:
Is appending `-cs` something standard?  If not, prefer we spell out `-catalog-source` or some other variant so it's easier to understand by a human.

Reply:
No, it isn't. I don't think it will stay that way (@rogbas is advocating moving to a monolithing registry image, which I like). Look for this PR to either get rehomed or shortly hereafter fundamentally changed.